### PR TITLE
Story3 keyboard noises

### PR DESCRIPTION
**Pull request by patrickkenyon**
Sourced sounds from freesound. Associated sounds to city divs and associated keyboard keys and click events. Created function for keypresses and to make game noises. 

files to be reviewed:

- js/audiojs
- whether correct ids classes and data attributes are correct with your original version.

### CODE REVIEW

**Review by f-gate — DISMISSED** [review]
Looks great

File: JS/audio.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var audioObj = {
+    cheer1: 'audio/cheer1.wav',
+    cheer2: 'audio/cheer2.wav',
+    cheer3: 'audio/cheer3.wav',
+    cheer4: 'audio/cheer4.wav'
+}
+
+function listenKeypress() {
+    document.addEventListener('keypress', function(e) {
+        switch (true) {
```
Comment (mporam) [inline JS/audio.js]: you could optimize this code using an object lookup:
```
var keys = {'q' : 'cheer1'} // etc
if (e.key in keys) {
    makeNoise(keys[e.key], audiObj)
}
```
```

File: JS/audio.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                makeNoise('cheer3', audioObj)
+                break
+            case (e.key === 'r'):
+                makeNoise('cheer4', audioObj)
+                break
+        }
+    })
+}
+
+listenKeypress()
```
Comment (mporam) [inline JS/audio.js:15]: why create a function to then call immediately?

File: JS/audio.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+var cities = document.querySelectorAll('.city')
+cities.forEach(function(city) {
+    city.addEventListener('click', function() {
+        var param = 'cheer' + city.dataset.city
+        makeNoise(param, audioObj)
+    })
+})
+
+function makeNoise(noise, audioObject) {
```
Comment (mporam) [inline JS/audio.js]: if you create the audioObject in hthe function you wont have to keep passing it in. you don't need it outside the function

**Review by mporam — CHANGES_REQUESTED** [review]

File: JS/audio.js (hunk 1 above)
Comment (ucanfil) [reply JS/audio.js]: This shortened our code a lot, thanks. 👍

File: JS/audio.js (hunk 2 above)
Comment (ucanfil) [reply JS/audio.js:15]: This is to run the function after game starts.

File: JS/audio.js (hunk 3 above)
Comment (ucanfil) [reply JS/audio.js]: audioObj moved inside the makeNoise function 👍

**Review by mporam — APPROVED** [review]

**Pull request closed without merging**